manifests/install.pp

```diff
@@ -33,12 +33,18 @@
     require => Exec["pyenv::checkout ${user}"],
   }
 
-  exec { "pyenv::shrc ${user}":
-    command => "echo 'source ${pyenvrc}' >> ${shrc}",
-    user    => $user,
+  file { $shrc:
```

Comment:
> @abloomston Your issue on travis is caused by the old spec since you removed the Exec[pyenv::shrc ${user}].

@Sliim apologies for missing this. I missed the [build error because it didn't mark the build as failed](https://travis-ci.org/abloomston/puppet-pyenv/jobs/194862812):

```
rake aborted!
KeyError: key{linenumber} not found
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/puppet-lint-2.1.0/lib/puppet-lint.rb:103:in `%'
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/puppet-lint-2.1.0/lib/puppet-lint.rb:103:in `format_message'
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/puppet-lint-2.1.0/lib/puppet-lint.rb:152:in `block in report'
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/puppet-lint-2.1.0/lib/puppet-lint.rb:142:in `each'
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/puppet-lint-2.1.0/lib/puppet-lint.rb:142:in `report'
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/puppet-lint-2.1.0/lib/puppet-lint.rb:201:in `print_problems'
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/puppet-lint-2.1.0/lib/puppet-lint/tasks/puppet-lint.rb:80:in `block (3 levels) in define'
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/puppet-lint-2.1.0/lib/puppet-lint/tasks/puppet-lint.rb:77:in `each'
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/puppet-lint-2.1.0/lib/puppet-lint/tasks/puppet-lint.rb:77:in `block (2 levels) in define'
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/puppet-lint-2.1.0/lib/puppet-lint/tasks/puppet-lint.rb:71:in `block in define'
/home/travis/.rvm/gems/ruby-2.0.0-p598/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
/home/travis/.rvm/gems/ruby-2.0.0-p598/bin/ruby_executable_hooks:15:in `eval'
/home/travis/.rvm/gems/ruby-2.0.0-p598/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => test => lint
(See full trace by running task with --trace)
```

I'm not familiar with rake, so am unclear if there might be a configuration change required in the Rakefile so that a failure in the call to `puppet-lint` is honored.